packages/@o3r/core/schematics/index.it.spec.ts

```diff
@@ -107,133 +103,4 @@ describe('new otter application', () => {
       }
     });
   });
-
-  describe('monorepo', () => {
```

Comment:
:'(

Reply:
Not removed, just moved to workspace package ;)

Reply:
I don't quite understand why you moved it, it almost exclusively tests `@o3r/core`

Reply:
It's actually something we can discuss and I can put it back to @o3r/core.
I moved it because, at it was testing feature in the context or workspace I was think it would be simpler have them in the package dedicated to workspace (when the @o3r/core is really generic and I would expect to have more and more e2e in the future).